test/Syntax/Outputs/round_trip_parse_gen.swift.withkinds

```diff
@@ -73,6 +73,7 @@ class C <MemberDeclBlock>{<MemberDeclListItem><FunctionDecl>
 
   func foo3<FunctionSignature><ParameterClause>() </ParameterClause></FunctionSignature><CodeBlock>{<SequenceExpr><DiscardAssignmentExpr>
     _ </DiscardAssignmentExpr><AssignmentExpr>= </AssignmentExpr><FunctionCallExpr><ArrayExpr>[<ArrayElement><TypeExpr><SimpleTypeIdentifier>Any</SimpleTypeIdentifier></TypeExpr></ArrayElement>]</ArrayExpr>()</FunctionCallExpr></SequenceExpr><SequenceExpr><DiscardAssignmentExpr>
+    _ </DiscardAssignmentExpr><AssignmentExpr>= </AssignmentExpr><MemberAccessExpr><TupleExpr>(<TupleExprElement><AttributedType><Attribute>@convention(c) </Attribute><FunctionType>(<TupleTypeElement><SimpleTypeIdentifier>Int</SimpleTypeIdentifier></TupleTypeElement>) -> <SimpleTypeIdentifier>Void</SimpleTypeIdentifier></FunctionType></AttributedType></TupleExprElement>)</TupleExpr>.self</MemberAccessExpr></SequenceExpr><SequenceExpr><DiscardAssignmentExpr>
```

Comment:
According to [ExprNodes.py](https://github.com/apple/swift/blob/c311e451951b28afacebae69107cef28e4af9e44/utils/gyb_syntax_support/ExprNodes.py#L256), the expression child of a `TupleExprElement` must be of kind `Expr`. But if I'm reading this XML correctly, would this attempt to put an `AttributedType` directly into the `TupleExprElement`, which wouldn't "fit" because an `AttributedType` isn't an `Expr`?

What would this look like when SwiftSyntax tries to parse it into the gybbed data types; i.e., how would I access the type?

Reply:
Ah sorry, the test result file is outdated (I forgot to `git add` I believe).

Reply:
Updated